ServerCore/Pages/Components/PresenceComponent.razor.cs

```diff
@@ -33,6 +33,9 @@ public partial class PresenceComponent : IAsyncDisposable
         [Parameter]
         public int PuzzleId { get; set; }
 
+        [Parameter]
+        public bool IsReadOnly { get; set; }
```

Comment:
I think this is a somewhat weird name for the property. Wondering if "IsListComponent" is clearer.

Reply:
Maybe add a comment for this? ReadOnly is a good concept for this (that it shows presence, but doesn't write it), but I didn't immediately get it either